packages/lightning-mock/src/api-dev.ts

```diff
@@ -177,14 +176,18 @@ const setupRestAPI = (
     app.enqueueRun(run);
 
     // triggering wakeup in all connected workers
-    if (isWakeUp)
+    if (new Boolean(ctx.query.wakeup)) {
```

Comment:
This will always be true no matter the string that's passed to wakeup. 
henc, `/run?wakeup=true`, `/run?wakeup=false`, `/run?wakeup=anything` will always make this evaluate true.

Reply:
Oh. wrong

Reply:
actually true. I was overthinking after typing.

Reply:
Yep - changed it so I can do `/run?wakeup` and save myself 5 characters of typing time

Reply:
Actually I should do `if ('wakeup' in ctx.query)` because that's really what I want

Reply:
Yeah exactly. else with `/run?wakeup` the Boolean constructor will make it false

Reply:
Oh I see what you mean :sweat_smile: